examples/CIB/ex5/ex_main.cpp

```diff
@@ -0,0 +1,8 @@
+#include "main_one_body.cpp"
```

Comment:
Can we put in a comment to discourage this construct (`#include`ing the `cpp` file) and a "TODO:" comment indicating that we should fix this someday?

Reply:
Oops, this is actually exactly in line with the way the other IBAMR examples are set up. :flushed:

I think we can clean this up in a later PR, after #494.